### PR TITLE
Accessibility fix: Make Learn more link more descriptive

### PR DESCRIPTION
**Pull request by lyndaidaii**
Updating the aria-label in this case to a more descriptive text is sufficient

Addresses  https://github.com/NuGet/NuGetGallery/issues/8184

### CODE REVIEW

File: src/NuGetGallery/Strings.resx [hunk 1]
```diff
@@ -1187,4 +1187,16 @@ The {1} Team</value>
   <data name="ReadmeNotEditableWithEmbeddedReadme" xml:space="preserve">
     <value>The readme is not editable with the package has the embedded readme</value>
   </data>
+  <data name="UploadPackage_LearMore_PackagingLicense" xml:space="preserve">
+    <value>Learn more about Packaging License within the nupkg</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx:1191]: /cc @jcjiang, could you review the working here?

File: src/NuGetGallery/Strings.resx [hunk 2]
```diff
@@ -1187,4 +1187,16 @@ The {1} Team</value>
   <data name="ReadmeNotEditableWithEmbeddedReadme" xml:space="preserve">
     <value>The readme is not editable with the package has the embedded readme</value>
   </data>
+  <data name="UploadPackage_LearMore_PackagingLicense" xml:space="preserve">
+    <value>Learn more about Packaging License within the nupkg</value>
+  </data>
+  <data name="UploadPackage_LearnMore_IconUrlDeprecation" xml:space="preserve">
+    <value>Learn more about Icon Url deprecation</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx:1194]:
```suggestion
    <value>Learn more about icon URL deprecation</value>

```

File: src/NuGetGallery/Strings.resx (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx:1191]:
```suggestion
    <value>Learn more about including a license within the package</value>

```

File: src/NuGetGallery/Strings.resx [hunk 3]
```diff
@@ -1187,4 +1187,16 @@ The {1} Team</value>
...
     <value>The readme is not editable with the package has the embedded readme</value>
   </data>
+  <data name="UploadPackage_LearMore_PackagingLicense" xml:space="preserve">
+    <value>Learn more about Packaging License within the nupkg</value>
+  </data>
+  <data name="UploadPackage_LearnMore_IconUrlDeprecation" xml:space="preserve">
+    <value>Learn more about Icon Url deprecation</value>
+  </data>
+  <data name="UploadPackage_LearnMore_InvalidLicenseUrlEncoding" xml:space="preserve">
+    <value>Learn more about Invalid License Url encoding</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx:1197]:
```suggestion
    <value>Learn more about invalid license URL encoding</value>

```

File: src/NuGetGallery/Strings.resx [hunk 4]
```diff
@@ -1187,4 +1187,16 @@ The {1} Team</value>
...
+  </data>
+  <data name="UploadPackage_LearnMore_LicenseUrlDreprecation" xml:space="preserve">
+    <value>Learn more about License Url deprecation</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx:1200]:
```suggestion
    <value>Learn more about license URL deprecation</value>

```

File: src/NuGetGallery/Strings.resx (hunk 1 above)
Comment (lyndaidaii) [reply src/NuGetGallery/Strings.resx:1191]: @jcjiang  Could you help review the wording?

**Review by loic-sharma — APPROVED** [review]

File: src/NuGetGallery/Strings.resx (hunk 1 above)
Comment (jcjiang) [reply src/NuGetGallery/Strings.resx:1191]: This looks good!

**Review by jcjiang — APPROVED** [review]

**Pull request merged**